lib/Handler.js

```diff
@@ -1,15 +1,37 @@
+let isColdStart = true;
```

Comment:
I might consider creating a simple singleton to handle this something like

```js
const COLD_START = new class ColdStart {
  constructor() {
    this.state = true;
  }
  valueOf() {
    if (this.state) {
      this.state = false;
      return true;
    }
    return false;
  }
}();

if (+COLD_START) {
  console.log("1");
}

if (+COLD_START) {
  // not executed
  console.log("2");
}
```

Reply:
I think in this case I'm going to opt for the current pattern. This is a fine pattern and probably better in some respects, but I worry it is a little less clear in intent.